codegen/models/project_foundation/project_foundation.xml

```diff
@@ -124,6 +124,9 @@
     <class name="key provider"/>
     <class name="signer"/>
     <class name="verifier"/>
+    <class name="simple swu"/>
```

Comment:
Why this stuff goes to the `foundation`?

Reply:
Because brain key depends on it

Reply:
May be a new library will be an appropriate place?

Reply:
I don't see point creating new library for only 1 simple (it's SIMPLE swu he-he) class

Reply:
Hashing to a point is a basic algorithm and might be needed in other protocols

Reply:
Ok. What about "brainkey client" and "brainkey server"?

Reply:
What's the point of moving them to separate library? Because cons are serious: that creates 2 additional frameworks for other languages

Reply:
Make JS libraries as small as possible.
@vadimavdeev

Reply:
@SergeySeroshtan separating classes into different libraries all of which you are going to link in the end, does not reduce size. We are going to replace massive Pythia + Relic with this 3 classes, that what size reduction is.

Reply:
Not for JS. There all libraries go separately.